docs/user_manual/components.md

```diff
@@ -181,6 +203,28 @@ It can happen that `tap_min > tap_max`. In this case the winding voltage is decr
 increased.
 ```
 
+#### Electric Model
+`transformer` is described by a π model, where $Z_{series}$ and $Y_{shunt}$ can be computed by following equations:
+
+$$
+   \begin{eqnarray} 
+        z_{base} = sn/(u2^2)\\
+        |Z_{series}| = uk/z_{base}\\ 
+        Re(Z_{series}) = (pk/sn)/z_{base},\\
+        Im(Z_{series}) = uk * \sqrt{|Z_{series}|^2-Re(Z_{series})^2},\\
+        base\_ i\_ to = base\_ power\_ 3p / u2 / sqrt(3),\\
+        base\_ y\_ to = base\_ i\_ to^2 / base\_ power\_ 1p,\\
+        Y_{series} = (1/Z_{seires}) / base\_ y\_ to,\\
+        |Y_{shunt}| = i0 * sn / u2^2\\
+        Re(Y_{shunt}) = p0 / (u2^2)\\
+        Im(Y_{shunt}) = 0 (if Re(Y_{shunt}) > |Y_{shunt}|) or = -sqrt(|Y_{shunt}|^2-Re(Y_{shunt})^2)\\
+        Y_{shunt} = Y_{shunt} / base\_ y\_ to\\
+   \end{eqnarray}
+$$
+
+where base_power_3p = $1 \times 10^6$, and base_power_1p = base_power_3p / 3. 
```

Comment:
```suggestion
$$
   \begin{eqnarray} 
        |Z_{series}| = uk p.u.\\ 
        Re(Z_{series}) = (pk/sn),\\
        Im(Z_{series}) = sqrt{|Z_{series}|^2-Re(Z_{series})^2},\\
        |Y_{shunt}| = i0 p.u. \\
        Re(Y_{shunt}) = sn/p0 p.u. \\
        Im(Y_{shunt}) = 0 (if Re(Y_{shunt}) > |Y_{shunt}|) or = -sqrt(|Y_{shunt}|^2-Re(Y_{shunt})^2) p.u. \\
   \end{eqnarray}
$$

where base_power_3p = $1 \times 10^6$, and base_power_1p = base_power_3p / 3. 
```

Need to confirm Re yshunt once

Reply:
Also add that the scaling of these parameters happen in a linear fashion for different tap positions

Reply:
Or better:

```suggestion
$$
   \begin{eqnarray} 
        |Z_{series}| = uk / z_{base}.\\ 
        Re(Z_{series}) = (pk/sn) / z_{base} ,\\
        Im(Z_{series}) = sqrt{|Z_{series}|^2-Re(Z_{series})^2} p.u. ,\\
        |Y_{shunt}| = i0 / y_{base} \\
        Re(Y_{shunt}) = (sn/p0) / y_{base} \\
        Im(Y_{shunt}) = 0 (if Re(Y_{shunt}) > |Y_{shunt}|) or = -sqrt(|Y_{shunt}|^2-Re(Y_{shunt})^2) \\
        z_{base} = sn/(u2^2)\\
        y_{base} = (u2^2)/sn\\
   \end{eqnarray}
$$

```